File: java/com/countrygamer/countrygamercore/common/CommonProxy.java
package com.countrygamer.countrygamercore.common;

import net.minecraft.entity.player.EntityPlayer;
import net.minecraft.world.World;

import com.countrygamer.countrygamercore.Base.Plugin.PluginCommonProxy;
import com.countrygamer.countrygamercore.Base.common.network.MessageSyncExtendedProperties;
import com.countrygamer.countrygamercore.Base.common.network.PacketHandler;

public class CommonProxy implements PluginCommonProxy {
	
	@Override
	public void registerRender() {
	}
	
	public void syncPacket(MessageSyncExtendedProperties message, EntityPlayer player) {
		PacketHandler.sendToPlayer(Core.pluginID, message, player);
	}
	
	@Override
	public Object getServerGuiElement(int ID, EntityPlayer player, World world,
			int x, int y, int z) {
		return null;
	}
	
	
	@Override
	public Object getClientGuiElement(int ID, EntityPlayer player, World world,
			int x, int y, int z) {
		return null;
	}
	
}

File: src/main/scala/com/countrygamer/cgo/common/CommonProxy.scala
package com.countrygamer.cgo.common

import com.countrygamer.cgo.common.network.{PacketHandler, PacketSyncExtendedProperties}
import com.countrygamer.cgo.wrapper.common.ProxyWrapper
import net.minecraft.entity.player.EntityPlayer
import net.minecraft.world.World

/**
 *
 *
 * @author CountryGamer
 */
class CommonProxy extends ProxyWrapper {

	override def registerRender(): Unit = {}

	override def getServerGuiElement(p1: Int, p2: EntityPlayer, p3: World, p4: Int, p5: Int,
			p6: Int): AnyRef = {
		null
	}

	override def getClientGuiElement(p1: Int, p2: EntityPlayer, p3: World, p4: Int, p5: Int,
			p6: Int): AnyRef = {
		null
	}

	def syncPacket(message: PacketSyncExtendedProperties, player: EntityPlayer) {
		PacketHandler.sendToPlayer(Origin.pluginID, message, player)
	}

	def addArmor(armor: String): Int = {
		0
	}

}
